.github/workflows/test-and-deploy.yml

```diff
@@ -4,7 +4,7 @@ on: [push, pull_request]
 
 jobs:
   test:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
For these we are currently actually deploying on ubuntu-20.04 so we cannot update these without further checking to ensure the built distribution doesn't (somehow) have some tendrils from 22.04 in it.